lib/lib_test.go

```diff
@@ -44,7 +48,7 @@ var _ = Describe("Handler", func() {
 		msg, _ := ioutil.ReadFile("../res/cert.json")
 		It("should return a valid payload", func() {
 			result, err := lib.ParseResultCertificate(msg)
-			slackPayload := lib.NewSlackPayload(config, result)
+			slackPayload := slack.NewSlackPayload(config, result)
```

Comment:
We should rename the method in `slack.NewPayload()`, what do you think?

Reply:
Yes, I updated `pkg/slack`.